black.py

```diff
@@ -1364,27 +1346,83 @@ def preceding_leaf(node: Optional[LN]) -> Optional[Leaf]:
     return None
 
 
-def is_delimiter(leaf: Leaf) -> int:
-    """Return the priority of the `leaf` delimiter. Return 0 if not delimiter.
+def is_split_after_delimiter(leaf: Leaf, previous: Leaf = None) -> int:
+    """Returns the priority of the `leaf` delimiter, given that it is a delimiter
```

Comment:
nit: Have you noticed how I modified all docstrings to be imperative? "Return" rather than "Returns".